core/core.mk

```diff
@@ -90,9 +90,21 @@ base-prefix :=
 
 CFG_CRYPTOLIB_NAME ?= tomcrypt
 CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
+
+ifeq ($(CFG_CRYPTOLIB_NAME), tomcrypt)
+CFG_CRYPTO_TOMCRYPT := y
+else ifeq ($(CFG_CRYPTOLIB_NAME), mbedtls)
+CFG_CRYPTO_MBEDTLS := y
+else
+$(error Error: $(WITH_CRYPTO) is not supported.)
+endif
+
 libname = $(CFG_CRYPTOLIB_NAME)
 libdir = $(CFG_CRYPTOLIB_DIR)
 include mk/lib.mk
+libname = tomcrypt_lite
+libdir = core/lib/libtomcrypt_lite
+include mk/lib.mk
```

Comment:
I think this should be dependent on CFG_CRYPTOLIB_NAME=mbedtls to avoid overlap with other crypto libs

Reply:
Now, the `libtomcrypt_lite` are needed by mbedtls and tomcrypt. I know you mean is that maybe another crypto lib will be added in future. So, I will try to distinguish them.